backend/bin/dev/dev.go

```diff
@@ -117,22 +118,22 @@ func tryRunServer(logger logging.Logger) error {
 		logger.Log("msg", "No Emailer selected")
 	}
 
-	mux := http.NewServeMux()
-
-	mux.Handle("/web/", http.StripPrefix("/web", server.Web(
-		db, contentStore, &server.WebConfig{
-			CSRFAuthKey:      []byte("DEVELOPMENT_CSRF_AUTH_KEY_SECRET"),
-			SessionStoreKey:  []byte("DEVELOPMENT_SESSION_STORE_KEY_SECRET"),
-			UseSecureCookies: false,
-			AuthSchemes:      schemes,
-			Logger:           logger,
-		},
-	)))
-	mux.Handle("/api/", server.API(
```

Comment:
/web/ and /api/ should both be ported to the new router for dev.go

Reply:
whoops! Just a second